crates/broker/src/config.rs

```diff
@@ -126,6 +126,14 @@ pub struct MarketConf {
     ///
     /// Maximum number of concurrent lockin requests that can be processed at once
     pub max_concurrent_locks: Option<u32>,
+    /// Optional cache directory for storing downloaded images and inputs
+    /// if not set, files will be re-downloaded every time
+    pub cache_dir: Option<PathBuf>,
+    /// Optional S3 endpoint URL
+    pub s3_endpoint_url: Option<String>,
```

Comment:
This is only needed for minio testing and unfortunately the AWS SDK config doesn't load it from ENV like all the other parameters.

Reply:
Mentioned in my other comment, but I believe there actually is an env variable for this.

Reply:
You are right, the `AWS_ENDPOINT_URL` env is loaded by default. Addressed in c467ce2c6c3bd0d85b10b50a14c862d940cdfa61